src/GenViz.jl

```diff
@@ -40,12 +40,17 @@ end
 
 struct VizServer
   visualizations::Dict{String, Viz}
+  host
   port
   connectionslock
 end
 
 function VizServer(port)
-    server = VizServer(Dict{String, Viz}(), port, Base.Threads.SpinLock())
+    VizServer("127.0.0.1", port)
+end
+
+function VizServer(host, port)
+    server = VizServer(Dict{String, Viz}(), host, port, Base.Threads.SpinLock())
     @async with_logger(NullLogger()) do
         HTTP.listen("0.0.0.0", port) do http
```

Comment:
I assume this `0.0.0.0` means that we will still get these requests even if the hostname is some other IP?

Reply:
`0.0.0.0` tells the server to listen on all interfaces not just loopback. That was the patch we needed last time I was working with the Docker image.